### ISSUE TITLE
Verify Grant Issue

### ISSUE THREAD

**Issue by katcita** [gitcoinco/web#8894]
### Who

Chemical engineer turned web dev in ruby/rails/js/html5. Working on https://gitcoin.co/grants/2521/eli5-crypto

### What 

For this fund https://gitcoin.co/grants/2521/eli5-crypto A fund to support my youtube channel where I explain how Ethereum and DeFi work with easy to understand aninations.

### Why

Making these videos take time and money so I'm looking for donations to help me keep going.

**Comment by gdixon** [comment]
Hi @katcita,

Can you verify your ownership via Twitter please, click 'Verify Grant' at the top right of your grants details page, and follow the given prompts.

I'm going to close out this issue, but please feel free to reopen if you run into any problems.
